types/async-rerun/index.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @param {Function} fn The function which will be used as parameter should return a PROMISE & SHOULD NOT HAVE ANY PARAMETER it's compulsory, as its a async retry mechanism. Also you can BIND the global scope too with the function in order use the global scoped variables like this => <function name>.bind(this).
+ * @param {number} maxRetries Max Retries in number.
+ * @param {number} delayInMS Delay time for retries in number as mili-seconds(ms).
+ * @param {boolean} alwaysRetry To retry always untill the positive response, If its true then maxRetries will not count.
+ * @param {boolean} showLogs To show logs for all retries.
+ */
```

Comment:
It looks like the JSDoc is for the wrong property (seems to be intended for `runAsync`)

Reply:
FYI type annotations in the JSDoc (such as `{Function}` or `{number}`) aren't needed in TypeScript, since TypeScript provides those hints

Reply:
Yeah, you were right!
Please review it again I just fixed it now.
Thanks